### ISSUE TITLE
safe cat -iii with xorurl exposes xorurl instead of subname

### ISSUE THREAD

**Issue by calumcraig** [maidsafe/safe-cli#207]

**Comment by bochaco** [comment]
Thanks @calumcraig . We'll need to have the public name as part of the NRS Map metadata, since when fetching an NRS Map Container using its XOR-URL we cannot know what's the corresponding public name. We could fix it now with our pesudo RDF, or wait till we have proper RDF data representation.

**Comment by calumcraig** [comment]
Yes, as discussed @bochaco, it's your call to fix it now or wait for the proper RDF implementation.
